benchpress/benchpress/plugins/hooks/toplev.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+import logging
+
+from benchpress.lib.hook import Hook
+from benchpress.lib import util
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_BACKGROUND_DURATION_SECS = '300'
```

Comment:
300s might be too long for some workloads like GAP which don't sustain steady state for that long. Maybe shorten the interval to 60s?

Reply:
Sure sounds good.